### PR TITLE
man: make clear that sd-journal notifications always come with extra …

### PR DESCRIPTION
**Pull request by poettering**
…latency

Replaces: #17699

### CODE REVIEW

File: man/sd_journal_get_fd.xml [hunk 1]
```diff
@@ -151,18 +151,20 @@ else {
...
-    to be polled for and hence might be noticed only with a certain
-    latency. Note that there is usually no need to invoke this function
-    directly as <function>sd_journal_get_timeout()</function> on these
-    file systems will ask for timeouts explicitly anyway.</para>
+    <para><function>sd_journal_reliable_fd()</function> may be used to check whether the wake-up events from
+    the file descriptor returned by <function>sd_journal_get_fd()</function> are known to be quickly
+    triggered. On certain file systems where file change events from the OS are not available (such as NFS)
+    changes need to be polled for repeatedly, and hence are detected only with a considerable latency. This
+    call will return a positive value if the journal changes are detected quickly and zero when they need to
+    be polled for and hence might be noticed only with a considerable latency. Note that there is usually no
```
Comment (keszybz) [inline man/sd_journal_get_fd.xml]: Drop " and hence might be noticed only with a considerable latency"? It's just repeating what was already said.

File: man/sd_journal_get_fd.xml [hunk 2]
```diff
@@ -151,18 +151,20 @@ else {
...
+    need to invoke this function directly as on these file systems
+    <function>sd_journal_get_timeout()</function> will request appropriate timeouts anyway.</para>
```
Comment (keszybz) [inline man/sd_journal_get_fd.xml:160]: Drop "as on these file systems"?

File: man/sd_journal_get_fd.xml [hunk 3]
```diff
@@ -151,18 +151,20 @@ else {
...
+
+    <para>Note that the all of the above change notification interfaces will typically not notice changes
```
Comment (keszybz) [inline man/sd_journal_get_fd.xml]: "Note that all of the above change notification interfaces do not report changes instantly. Latencies are introduced for multiple reasons: as mentioned certain storage backends require time-based polling, in other cases wake-ups are optimized by coalescing events, and the OS introduces additional IO/CPU scheduling latencies." ?

**Review by keszybz — COMMENTED** [review]
Looks good, but suggestion for the text.

**Comment by poettering** [comment]
Force pushed a new version. Made the suggested changes 1:1, hence upgrading green label.

**Pull request merged**